src/index.js

```diff
@@ -43,8 +43,8 @@ function EvolvClient(options) {
 
   const context = new Context();
   const store = new Store(options);
-  const contextBeacon = options.analytics ? new Beacon(options.endpoint + '/' + options.environment + '/analytics') : null;
```

Comment:
Nit: We should consider leaving the name for this endpoint "analytics". The spec calls it "data" but that is because in V2, it merges the events and analytics into a single endpoint where all "data" is sent.